internal/resource/config/administrativeaccount/administrative_account_resource.go

```diff
@@ -75,49 +75,52 @@ func (r *administrativeAccountsResource) Schema(ctx context.Context, req resourc
 				Optional:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"description": schema.StringAttribute{
 				Description: "Description of the account.",
 				Optional:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"email_address": schema.StringAttribute{
 				Description: "Email address associated with the account.",
 				Optional:    true,
-				Computed:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"password": schema.StringAttribute{
-				Description: "Password for the Account. This field is only applicable during a POST operation.",
-				Required:    true,
+				Description: "Password for the Account. This field is only applicable during account creation.",
+				Computed:    true,
+				Optional:    true,
```

Comment:
Can we just make this required? It is required during creation, right?

Reply:
This is now required